projects/stackoverflow/src/stackoverflow/DataExtractor.java

```diff
@@ -0,0 +1,178 @@
+package stackoverflow;
```

Comment:
Please place this file directly under `projects/stackoverflow`, and change the package to `projects.stackoverflow`.